charts/github-actions-runners/values.yaml

```diff
@@ -0,0 +1,35 @@
+# -- Declare the name of the runner deployment
+name: "github-runner-deployment"
+# -- Select Deploment type - [ repository / organization / enterprise ]
+deploymentType: "repository"
+# -- For use with repository deployment type - Add repo for runner deployment
+repositoryName: "example/repo"
+# -- For use with organization deployment type
+organizationName: ""
+# -- For use with enterprise deployment type
+enterpriseName: ""
+# Declare the minimum runner scale down time after scale out (Default :  5 minutes).
+scaleDownDelaySecondsAfterScaleOut: "300"
+# -- Declare the minimum and maximum available runners
+minimumReplicaCount: "1"
+maximumReplicaCount: "1"
+# -- Maximum time that a runner should exist after scale up activity without an assigned workflow.
+scaleUpTriggerDuration: "10m"
+# -- Define the image for the runners to be deployed with.
+image:
```

Comment:
nit: image is typically the first item in a helm chart

Reply:
Is this a standard of helm or just a community convention Joe?

Reply:
Just a community convention I've noticed, helm has very minimal official standards for things like this (https://helm.sh/docs/chart_best_practices/values/).
I think it can help slightly with readability to start with the application itself rather than underlying configuration, but definitely not a requirement.

Reply:
Suggested Changes made - Image now the first value 👍🏻